README.md

```diff
@@ -2,6 +2,33 @@
 
 UpSwyng is a mobile-ready, digital directory of resources to assist the unhoused and at-risk communities. This is a [Code For Boulder](https://www.codeforboulder.org) project coming soon to Boulder County, Colorado.
 
+## Before you begin
+
+You can save yourself some grief and aggravation by downloading the versions of the following programs as specified in https://github.com/CodeForBoulder/upswyng/blob/master/package.json
+You will need to check that you have each of the following using the which command or add -v or --version to the name of the program:
+
+0. xcode ==> ``` /usr/bin/xcodebuild -version ```
```

Comment:
So this is listed for the native version, ya? The problem is that we don't advertise the native version much since it's out of date and not a priority feature. That being said, the readme for the native version is sorely out-of-date so perhaps something like this should live in that README for now.

Reply:
Nope you need it for homebrew if I’m not mistaken.

Reply:
It doesn't look like xcode is a requirement on the Home Brew site 🤔  I think we're safe to remove this as a required program.